cli/dcoscli/data/help/package.txt

```diff
@@ -56,6 +57,8 @@ Options:
         Command line only.
     --config
         Print the configurable properties of the `marathon.json` file.
+    --dry-run, -n
```

Comment:
Oh well I put it there not so much to protect against failure but to protect against success. I'm testing this against a DCOS cluster that I use for other things and I didn't want to risk adding some repos that don't belong and messing something up.

But that was largely because I had messed up my https://github.com/dcos/dcos-docker environment, but now I got it working again. Speaking of which, is that what you'd recommend for development. I looked at dcos-vagrant and dcos-docker and dcos-docker seems a bit nicer to me. dcos-docker hasn't had a commit in a while so I was curious if it's still active; I sent a few PRs to it.

I think I'll just remove the `--dry-run` thing, since it was more of a development thing and probably not too useful in normal usage.